src/data/store/find.js

```diff
@@ -48,6 +69,37 @@ function findUtil(url, Model, dataQuery, asyncHandler) {
     url += '?' + query.join('&')
   }
 
+  return url
+}
+
+export function findUtil(dataQuery, asyncHandler) {
+  const searchLocally = shouldSearchLocally.call(this, dataQuery)
+  const localStoragePolicy = getLocalStoragePolicy.call(this, dataQuery)
+  const shouldStoreResult = Utils.isBrowser && !searchLocally && localStoragePolicy !== LocalStoragePolicy.DONOTSTOREANY
```

Comment:
does it make sense to create a new util method `Utils.isOfflineAvailable`?

Reply:
It will be the same as Utils.isBrowser